### DISCUSSION TITLE
Tracking focus loss on LineEdit

### DISCUSSION THREAD

**Discussion by maxenko** [slint-ui/slint#5373] [Q&A]
How can I track when user has clicked off / tabbed over from a line edit (causing focus to leave input field) and moved on to using other widgets.

Is this possible in slint?

**Comment by ghost** [comment]
Hi,
currently this is only possible for the `FocusScope` using the `focus-changed-event()` callback. But there is a ongoing task to implement property changed callbacks. It is still part of the code base but marked as experimental. So soon something like this would be possible:

```slint
LineEdit {
    changed has-focus => {
        debug("Focus changed");
    }
}
```
